.github/dependabot.yml

```diff
@@ -167,4 +203,8 @@ updates:
       time: "04:20"
       timezone: Europe/Paris
     open-pull-requests-limit: 5
+    groups:
```

Comment:
Might want to insert it before `open-pull-requests-limit` for consistency ^^

Reply:
Indeed, I've just changed that